pkg/polaris/graphql/core/sla_test.go

```diff
@@ -0,0 +1,166 @@
+package core
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"io"
+	"net/http"
+	"reflect"
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/rubrikinc/rubrik-polaris-sdk-for-go/internal/testnet"
+	"github.com/rubrikinc/rubrik-polaris-sdk-for-go/pkg/polaris/graphql"
+	"github.com/rubrikinc/rubrik-polaris-sdk-for-go/pkg/polaris/log"
+)
+
+func TestAssignSLAForSnappableHierarchies(t *testing.T) {
+
+	objIDs := []uuid.UUID{uuid.New()}
+
+	testClient := func(httpResp error, gResp bool) (
+		*graphql.Client,
+		*http.Server,
+	) {
+		client, lis := graphql.NewTestClient("john", "doe", log.DiscardLogger{})
+		var srv *http.Server
+		defer func(srv *http.Server) {
+			if srv != nil {
+				_ = srv.Shutdown(context.Background())
+			}
+		}(srv)
```

Comment:
Looks like this code is duplicated, there is another closure down in the test function, at line 132-134.